### PR TITLE
Create recognizing-the-dhamma_geoff.md

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/essays/recognizing-the-dhamma_geoff.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Recognizing the dhamma: a study guide"
```
Comment (buddhist-uni) [inline _content/essays/recognizing-the-dhamma_geoff.md:2]: capitalization

File: _content/essays/recognizing-the-dhamma_geoff.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/Archive/Writings/Ebooks/RecognizingtheDhamma_181215.pdf"
```
Comment (buddhist-uni) [inline _content/essays/recognizing-the-dhamma_geoff.md:5]: Use the proper share url: https://www.dhammatalks.org/ebook_index.html#RecognizingTheDhamma

Also: add the pdf and epub drive_links

File: _content/essays/recognizing-the-dhamma_geoff.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+course: thought
+tags:
+  - suttas
+  - fetters
+year: 2013 
+pages: 103
```
Comment (buddhist-uni) [inline _content/essays/recognizing-the-dhamma_geoff.md:11]: This belongs in `booklets` not `essays`

File: _content/essays/recognizing-the-dhamma_geoff.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Recognizing the dhamma: a study guide"
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/Archive/Writings/Ebooks/RecognizingtheDhamma_181215.pdf"
+course: thought
+tags:
+  - suttas
```
Comment (buddhist-uni) [inline _content/essays/recognizing-the-dhamma_geoff.md:8]: the tag is `sutta` singular, not plural

**Review by buddhist-uni — CHANGES_REQUESTED** [review]
Try again. 😉

**Comment by JJM317** [comment]
I capitalized the 'a' after the colon. I put the Drive links for both pdf
and epub. What's the issue?

Kindly,
~Joe

File: _content/booklets/recognizing-the-dhamma_geoff.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Recognizing the Dhamma: A study guide"
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/Archive/Writings/Ebooks/RecognizingtheDhamma_181215.pdf"
```
Comment (buddhist-uni) [inline _content/booklets/recognizing-the-dhamma_geoff.md:5]: Please use https://www.dhammatalks.org/ebook_index.html#RecognizingTheDhamma for the external url

File: _content/booklets/recognizing-the-dhamma_geoff.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Recognizing the Dhamma: A study guide"
```
Comment (buddhist-uni) [inline _content/booklets/recognizing-the-dhamma_geoff.md:2]: Shouldn't study guide also be in title case?

File: _content/booklets/recognizing-the-dhamma_geoff.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/Archive/Writings/Ebooks/RecognizingtheDhamma_181215.pdf"
+drive_url: "https://drive.google.com/file/d/1TCvWjpFNN4fKfne6Ja-S0taRGpRNPTCu/view?usp=drive_link"
+drive_url: :https://drive.google.com/file/d/15S-Oj6mPsSRw004kfbeISUPm3sov4bMn/view?usp=drive_link"
```
Comment (buddhist-uni) [inline _content/booklets/recognizing-the-dhamma_geoff.md:7]: this is not the right format for `drive_links`

**Comment by JJM317** [comment]
So, APA style is just the first word after a colon is capitalized. MLA and
CMS capitalize everything except prepositions. For some reason, I've had
APA in mind (minus that I missed the initial A by mistake). I'll start
making sure to follow the MLS/CMS style.
--------------------

**Review by buddhist-uni — COMMENTED** [review]
Okay... And the drive links? And the external url?

**Comment by JJM317** [comment]
What's the problem with them? I used the pdf and epub links from Drive. The
external link is from the website - is the pdf url a problem? Should we
talk about it next Wednesday?
Kindly,
~Joe

**Pull request merged**